orionsdk/solarwinds.py

```diff
@@ -834,3 +875,19 @@ def ncm_run_compliance_report(self, report_name):
         report_id = results['results'][0]['PolicyReportID']
         self.swis.invoke('Cirrus.PolicyReports', 'StartCaching', [report_id])
 
+    def add_dns_a_record_with_ptr_to_ipam(self, name, ip_address, domain, server):
```

Comment:
For consistency, I'd keep the parameter order the same here as in the invocation below: server then domain.